src/components/Activities/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { ReactNode } from 'react'
+import NextLink from 'next/link'
+import { Box, Link, Stack, SvgIcon, Typography } from '@mui/material'
+import { AppRoutes } from '@/config/routes'
+import PaperContainer from '@/components//PaperContainer'
+import { ExternalLink } from '@/components/ExternalLink'
+import { ChevronLeft } from '@mui/icons-material'
+import SafePass from '@/public/images/safe-pass.svg'
+import UserIcon from '@/public/images/user.svg'
+import TransactionsVolumeIcon from '@/public/images/transactions-volume.svg'
+import TransactionsNumberIcon from '@/public/images/transactions-number.svg'
+import AssetsStoredIcon from '@/public/images/assets-stored.svg'
+import EmptyActivityIcon from '@/public/images/empty-activity.png'
+import ChatBubble from '@/public/images/chatbubble.png'
+import Image from 'next/image'
+
+const ProgramItem = ({ title, description, icon }: { title: string; description: ReactNode; icon: ReactNode }) => {
+  return (
+    <Box
+      textAlign="center"
+      display="flex"
+      flex="1"
+      flexDirection="column"
+      alignItems="center"
+      maxWidth="310px"
+      padding={5}
+    >
+      {icon}
+      <Typography variant="subtitle1" fontWeight="bold" mb={1}>
+        {title}
+      </Typography>
+      <Typography variant="body2" color="text.secondary" alignItems="center">
+        {description}
+      </Typography>
+    </Box>
+  )
+}
+
+const Activities = () => {
+  return (
+    <Box maxWidth="888px">
+      <Stack spacing={3}>
+        <Link
+          href={AppRoutes.activity}
+          component={NextLink}
+          sx={{ display: 'flex', alignItems: 'center', color: ({ palette }) => palette.primary.main }}
+        >
+          <ChevronLeft />
+          Back to main
+        </Link>
+
+        <Typography variant="h1">Eligible activities</Typography>
+        <PaperContainer>
+          <Stack alignItems="center" gap={2} pt={6}>
+            <Image src={ChatBubble} alt="Chat bubble" />
+
+            <SvgIcon component={SafePass} inheritViewBox sx={{ width: '282px', height: 'auto', my: 2 }} />
+
+            <Typography width="80%" textAlign="center">
+              With the{' '}
+              <Typography fontWeight="bold" display="inline">
+                {'Safe {Pass} Program'}
+              </Typography>{' '}
+              you can earn Points by using your Safe Account. Some activities are rewarded throughout the entire season,
+              some activities are only rewarded temporarily.
+            </Typography>
+
+            <ExternalLink href="">Learn more</ExternalLink>
```

Comment:
@schmanu Do we have content for this?

Reply:
@jmealy I think this is supposed to link to our landing page. But I am not sure. Best to ask in the Slack thread.